tests/OSPSuite.FuncParser.Tests/FuncParserSpecs.cs

```diff
@@ -3,7 +3,7 @@
 using OSPSuite.BDDHelper;
 using OSPSuite.BDDHelper.Extensions;
 
-namespace OSPSuite.FuncParser.FuncParserTests
+namespace OSPSuite.FuncParser
```

Comment:
same here: would keep namespace AS IS

Reply:
This is to match what we do everywhere else. I'd like our code to be consistent throughout the platform

Reply:
if some "inconsistency" helps to make a tests organization much more clear - for me it's a valid argument to use it.
"Consistency" is not a self purpose

Reply:
Agreed. I don't understand why having the same namespace makes things harder. Can you explain?

Reply:
ok gosh. I did not realize that you want to have each tests in its own namespace!! WHAT?!?!

Reply:
> Agreed. I don't understand why having the same namespace makes things harder. Can you explain?

Because having all tests in one namespace I just see plain list of all tests like this: 
![grafik](https://user-images.githubusercontent.com/25061876/61885044-79818100-aefd-11e9-9db4-d3228117d366.png)

> I did not realize that you want to have each tests in its own namespace

Not each test, but split the tests into test areas.
E.g. if I screw up the _dimension_ parsing (but the _formula_ parsing itself works fine)-I would immediately see it.

Reply:
reverted. Eventually, you'll have to explain why you don't use resharper :)